trade-finance-manager-ui/templates/deals/deals.njk

```diff
@@ -3,9 +3,9 @@
 
 {% extends "index.njk" %}
 
-{% block pageTitle %}
-  Trade Finance Manager
-{% endblock %}
+{% block pageTitle -%}
+  {{ heading }}
```

Comment:
This can be `{{- heading -}}` for a variable.

Reply:
Added extra space trimming.